src/Home.jsx

```diff
@@ -39,22 +39,33 @@ const Home = ({ classes }) => (
       root: classes.root,
     }}
     title="Firefox health"
-    subtitle="Tracking metrics for Firefox products">
+    subtitle="Tracking metrics for Firefox products"
+  >
     <div className={classes.links}>
       <Link to="/android">
-        <AndroidIcon /> Android
+        <AndroidIcon />
+        {' '}
+Android
```

Comment:
What rule does this!  It is ugly!

Reply:
It is! This was an automatic fix.
Want me to fix it before merging? Maybe a `span` is needed? I assume that white space is intended.